doc/api/report.md

```diff
@@ -131,11 +131,20 @@ is provided below for reference.
     }
   ],
   "javascriptHeap": {
-    "totalMemory": 6127616,
-    "totalCommittedMemory": 4357352,
-    "usedMemory": 3221136,
-    "availableMemory": 1521370240,
-    "memoryLimit": 1526909922,
+    "totalMemory": 5660672,
+    "executableMemory": 524288,
+    "totalCommittedMemory": 5488640,
+    "availableMemory": 4341379928,
+    "totalGlobalHandlesMemory": 8192,
+    "usedGlobalHandlesMemory": 3136,
+    "usedMemory": 4816432,
+    "memoryLimit": 4345298944,
+    "mallocedMemory": 254128,
+    "externalMemory": 315644,
+    "peakMallocedMemory": 98752,
+    "nativeContextCount": 1,
+    "detachedContextCount": 0,
+    "doesZapGarbage": 0,
```

Comment:
I'm not sure what the policies are on removing keys in the diagnostic report. When V8 removes these keys, what can we do to generate the report to fill these keys?

Reply:
These fields are currently following the V8 API, if V8 removes these fields, I think we need to change the value of the field to -1.

Reply:
It might be a good idea to add some comments here explaining about this (it's probably necessary to turn the `json` label to `js` for that, but it's probably also good to allow more comments on this example in general)

Reply:
> It might be a good idea to add some comments here explaining about this (it's probably necessary to turn the `json` label to `js` for that, but it's probably also good to allow more comments on this example in general)

Thanks. I have changed the format to `JS` and added `see v8.getHeapStatistics` to explain the `javascriptHeap` field.

Reply:
> It might be a good idea to add some comments here explaining about this (it's probably necessary to turn the `json` label to `js` for that, but it's probably also good to allow more comments on this example in general)

There are too many fields in this object, I need some time to understand the specific meaning of it, maybe submit another PR?